ui/src/test/Docs.test.js

```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react';
+import Docs from '../Pages/Docs/Docs';
+import {BrowserRouter as Router} from 'react-router-dom';
+
+describe('<Docs/> component', () => {
+    it('component should render',() => {
+        render(<Router><Docs /></Router>);
```

Comment:
I'm not sure what purpose wrapping server. Mind explaining?

Reply:
Good point. Removed from one test-case. Will add a new issue to update rest of the test-cases